### ISSUE TITLE
Why I cannot login into Observable? 

### ISSUE THREAD

**Issue by flyingduck92** [observablehq/feedback#508]
Dear Observable team,

I’ve been off the platform for almost a year. But when I want to login using my Github I keep getting this message pop out
My username previously was `@sekti92`
Also I tried to Sign Up again but still give the same notification

`The Observable account associated with this login is inactive.`
![image](https://user-images.githubusercontent.com/14885417/204009904-79092190-54e6-4270-aec1-fc258e1b670d.png)

Does anyone here can reach out the Observable Support Team Member?
I already sent a DM on Twitter and personally DM Mike Bostock on Twitter but got no answer.
Even I tried to email Observable Support at [support@observablehq.com](mailto:support@observablehq.com), but still got no answer.
Please help so I can get into my account again.
Thanks

**Comment by CobusT** [comment]
Hi. From looking at your account, I see that you did have an account @sekti92 and it used GitHub to authenticate. But it looks like it was deleted yesterday  at 2022-11-24 13:33:05 UTC. Did you perhaps already request that account to be deleted yesterday?

**Comment by flyingduck92** [comment]
@CobusT I did request to delete an account. But after that suddenly I got an invoice that going to charged me about $144. 
I was not active on the platform for a year. 
I did register for FREE Tier during my Internship, but after I make a request to delete an account yesterday, suddenly said on email I was a PRO member and saying going to charged me. That's why I tried to login again to checked again but always failed to login

Will I get charged $144 every year since my account already deleted and cannot login? Or do I still able to login and change it to FREE Tier? 
Thanks
